server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -405,6 +406,8 @@ public String executor() {
             } else {
                 throw new RuntimeException(e.getCause());
             }
+        } catch (ConnectTransportException ex) {
+            logger.info("No response from extension to request.");
```

Comment:
This can be written as:

```java
            } catch (CompletionException | ConnectTransportException e) {
            if (e.getCause() instanceof TimeoutException || e.getCause() instanceof ConnectTransportException) {
                logger.info("No response from extension to request.");
            }
            if (e.getCause() instanceof RuntimeException) {
                throw (RuntimeException) e.getCause();
            } else if (e.getCause() instanceof Error) {
                throw (Error) e.getCause();
            } else {
                throw new RuntimeException(e.getCause());
            }
       
```

Reply:
Plus we can add Exception name too in log to determine why didnt we get response. Just for sake of debugging..

Reply:
I got rid of the thrown exceptions too because OpenSearch will still fail if the exceptions are thrown